src/collection/components/modals/CollectionStillsModal.tsx

```diff
@@ -30,7 +33,7 @@ const CollectionStillsModal: FunctionComponent<CollectionStillsModalProps> = ({
 	isOpen,
 	collection,
 }) => {
-	const [videoStills, setVideoStills] = useState<string[] | null>(null);
+	const [videoStills, setVideoStills] = useState<string[]>();
```

Comment:
als je niet meegeeft moet het type ook undefined bevatten. De null die er stond maakte het expliciet, dat de initiële value niet ingesteld is.

Reply:
Fixed, maar vraag me wel af waarom tslint daar geen miserie over maakt. Omdat het perfect werkte, ging ik ervan uit dat dit zogezegd de initial useState value nog niet typed bekijkt maar vanaf dan enkel de types toelaat in de set, snap je?

Reply:
Ja goeie vraag. Mss kan de typescript compiler niet zover meerekenen en veronderstelt hij gewoon dat de initial value toch een string[] is zelfs als het eigenlijk initieel undefined is